### ISSUE TITLE
Enhancement: size of the invalid state

### ISSUE THREAD

**Issue by amaury-hanser** [PrestaShop/prestashop-ui-kit#213]
Hello, 

I suggest to increase the `font-size` of the validation message.
For now, the `font-size` compute to `10px`.

It comes from the mixin `@mixin ps-form-validation-state($state, $color)` in `scss/mixins/_forms.scss`
```
@mixin ps-form-validation-state($state, $color) {
  .form-control.is-#{$state},
  .is-#{$state} {
    border-color: $color;

    &:focus {
      box-shadow: none;
    }
  }

  .#{$state}-feedback {
    margin-top: 0.3125rem;
    font-size: $font-size-xs;
    font-weight: $font-weight-bold;
    color: $color;
  }
}
```
The variable `$font-size-xs` is defined in `scss/_variables.scss`: 
The value is:
```
$font-size-xs: 0.625rem; // PS custom
```

What do you think about increasing the computed value to `12px` instead?

**Comment by prestascott** [comment]
I agree with @amaury-hanser, `10px` is very small and difficult to read for users. 
The minimum size is `14px` but it's not the case in the back office. 
Size `12px` seems to be good according to the back office.
